.snyk

```diff
@@ -0,0 +1,2 @@
+language-settings:
```

Comment:
Snyk uses python 3.7.4 by default, but that version of python does not support Pillow 10.0.1
So this .snyk file should update the version of python that Snyk uses to validate our dependencies 

Supporting document here: Document here: https://docs.snyk.io/scan-using-snyk/supported-languages-and-frameworks/python

Reply:
Had a conversation with the security team, and they said it is fine to try to see if it fixes the issue. We will know once it's merged in!

Reply:
I will merge now and we will see